docs/developers_guide/micro_frontends_in_open_edx.rst

```diff
@@ -0,0 +1 @@
+This content has permanently moved to :doc:`Open edX Micro-frontend Developer's Guide <../micro_frontends/index>`.
```

Comment:
I wonder if our docs will get out of hand if we always leave the old docs. I meant we could add a redirect rile in readthedocs admin.

Note: we don’t have anything documented around leaving this kind of trail, so feel free to discuss as a topic or do what you feel is right.